packages/story-editor/src/components/canvas/editLayer.js

```diff
@@ -79,7 +81,10 @@ function EditLayer() {
 function EditLayerForElement({ element, showOverflow }) {
   const ref = useRef(null);
   const pageAreaRef = useRef(null);
-  const { editModeGrayout, EditMenu } = getDefinitionForType(element.type);
+  const { editModeGrayout } = getDefinitionForType(element.type);
+  const { isTrimMode } = useVideoTrim(({ state: { isTrimMode } }) => ({
```

Comment:
We could here return the trim mode directly, e.g.
`const isTrimMode = useVideoTrim(({ state }) => state.isTrimMode)`

(Even though I'm not sure why this file is changed)

Reply:
> (Even though I'm not sure why this file is changed)

See https://github.com/GoogleForCreators/web-stories-wp/pull/10452/files/f816101a70ebefbbef9e482ed695d0bf96361dea#r820758283

This line was copied 1:1 from `VideoEdit`